configure.ac

```diff
@@ -1,7 +1,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.69])
-AC_INIT([libdfp],[1.0.12],[tulioqm@br.ibm.com])
+AC_INIT([libdfp],[1.0.12],[murphyp@linux.vnet.ibm.com])
```

Comment:
I suggested that, because that information is used to set AC_PACKAGE_BUGREPORT, which is defined as:

    AC_PACKAGE_BUGREPORT, PACKAGE_BUGREPORT
        Exactly bug-report, if one was provided. Typically an email address, or URL to a bug management web page.